plugin/src/kiali/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -98,11 +102,24 @@ const expandableSectionStyle = kialiStyle({
 const nodeInfoStyle = kialiStyle({
   display: 'flex',
   alignItems: 'center',
-  marginTop: '0.25rem'
+  // marginTop: '0.25rem'
 });
 
 const workloadExpandableSectionStyle = classes(expandableSectionStyle, kialiStyle({ display: 'inline' }));
 
+// Helper function to check if Network Observability plugin is available and functional
+const isNetobservAvailable = (): boolean => {
+  return !!(
```

Comment:
```suggestion
  return (
```
this will already return a boolean value